library/ssl_tls.c

```diff
@@ -41,6 +41,7 @@
 #include "ssl_client.h"
 #include "ssl_debug_helpers.h"
 #include "ssl_misc.h"
+#include "legacy_or_psa.h"
```

Comment:
This must be included after `psa/crypto.h` to have symbols from `config_psa.h`.

Reply:
It is :) 
![image](https://user-images.githubusercontent.com/29195988/184912797-932dcf6f-8f66-4d2f-9ca1-b7c878aeec85.png)